build.gradle.kts

```diff
@@ -140,6 +142,31 @@ subprojects {
 		}
 
 		afterEvaluate {
+			if (project != project(":junit-jupiter")) {
```

Comment:
Why is this in an `afterEvaluate` block?

Reply:
Because that was the only place I could get  `${project.description}` to resolve correctly. Outside of this I was getting null. Any suggestions to work around that would be appreciated.

Reply:
this was also solved.